Sources/Application/Environment/AppEnvironment.swift

```diff
@@ -47,117 +76,122 @@ public final class AppEnvironment {
         }
         return string
     }
-    
-    static func updateReleaseFlags(_ flags: [String: Bool]) {
-        releaseFlags.flags = flags
-    }
 }
 
-class ReleaseFlags: FeatureFlagProvider {
-    var flags: [String: Bool] = [:]
-
-    subscript(key: String) -> Any? {
-        flags[key]
-    }
-}
-
-// MARK: - One Login Info Plist values as Type properties
+// MARK: - Mobile Back End Info Plist values as Type properties
 
 extension AppEnvironment {
-    static var oneLoginAuthorize: URL {
+    static var mobileBaseURLString: String {
+        string(for: .mobileBaseURL)
+    }
+    
+    static var mobileBaseURL: URL {
         var components = URLComponents()
         components.scheme = "https"
-        components.host = string(for: .oneLoginAuthorizeURL)
-        components.path = "/authorize"
+        components.host = mobileBaseURLString
         return components.url!
     }
     
-    static var oneLoginToken: URL {
+    static var mobileRedirect: URL {
         var components = URLComponents()
         components.scheme = "https"
-        components.host = string(for: .baseURL)
-        components.path = "/token"
+        components.host = mobileBaseURLString
+        components.path = "/redirect"
         return components.url!
     }
-
-    static var privacyPolicyURL: URL {
+    
+    static var walletCredentialIssuer: URL {
         var components = URLComponents()
         components.scheme = "https"
-        components.host = string(for: .externalBaseURL)
-        components.query = "lng=\(isLocaleWelsh ? "cy" : "en")"
```

Comment:
does this mean the credential issuer doesn't add the welsh parameter anymore?

Reply:
Sorry this file has be rearranged a bit now, you're looking at the privacy policy URL here which remains but is further down in this file 🙂 
The credential issuer never had the welsh parameter and still doesn't 🙂